engine/xml-config/src/test/java/org/datacleaner/configuration/DatastoreXmlExternalizerTest.java

```diff
@@ -108,11 +101,12 @@ public void testIsExternalizableAndExternalize() throws Exception {
         final String str = transform(elem);
 
         final char sep = File.separatorChar;
-        assertEquals("<csv-datastore name=\"foo\">" + "<filename>src" + sep + "test" + sep + "resources" + sep
-                + "example-dates.csv</filename><quote-char>\"</quote-char>"
-                + "<separator-char>,</separator-char><escape-char>\\</escape-char><encoding>UTF-8</encoding>"
-                + "<fail-on-inconsistencies>true</fail-on-inconsistencies><multiline-values>true</multiline-values>"
-                + "<header-line-number>1</header-line-number></csv-datastore>", str);
+        assertEquals("<csv-datastore name=\"foo\">\n" + "  <filename>src" + sep + "test" + sep + "resources" + sep
```

Comment:
Speaking of XPath. A Hamcrest XPath matcher would a much more understandable test than this... String nightmare :)